ReactAndroid/src/main/java/com/facebook/react/uimanager/UIManagerModuleConstants.java

```diff
@@ -81,7 +81,7 @@
         .build();
   }
 
-  public static Map<String, Object> getConstants() {
+  public static Map<String, Object> getConstants(float fontScale) {
```

Comment:
Didn't we decide we can get this off of displayMetrics as scaledDensity / density? At the least, I'd rather just pass in the Configuration object if you're worried about loss of precision